image_data_guide/03a_builtin_preprocessing.ipynb

```diff
@@ -0,0 +1,600 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Preprocessing Images for Built-in Algorithms (Part 3/4)\n",
+    "Download | Structure | **Preprocess** | Train  \n",
+    "\n",
+    "\n",
+    "**Note**: this notebook should be used with the conda_amazonei_mxnet_p36 kernel"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Overview\n",
+    "* #### Dependencies\n",
+    "* #### Application/x-image\n",
+    "* #### Application/x-recordio\n",
+    "* #### Upload to S3"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<pre>\n",
+    "</pre>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Dependencies\n",
+    "___\n",
+    "For this guide we'll use the SageMaker Python SDK version 2.9.2. By default, SageMaker Notebooks come with version 1.72.0. Other guides provided by Amazon may be set up to work with other versions of the Python SDK so you may wish to roll-back to 1.72.0."
```

Comment:
Studio was just updated sagemaker==2.15.0.
I tried this version and it works with this notebook.... so this upgrade can be removed.

Reply:
code cells updating the sagemaker sdk have been removed